src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people
+    .filter(person => century === undefined
+      ? person.sex === 'm'
```

Comment:
Consider creating a variable with a meaningful name and reusing it in your function instead of magic strings
```suggestion
      const MAN = 'm';
      
      person.sex === MAN
```